cpp/examples/basic/CMakeLists.txt

```
@@ -0,0 +1,34 @@
+cmake_minimum_required(VERSION 3.18)
+
+project(basic_example VERSION 0.0.1 LANGUAGES C CXX CUDA)
+
+set(CMAKE_CXX_STANDARD 14)
```

This should be 17 now I guess?
```suggestion
set(CMAKE_CXX_STANDARD 17)
```